components/Dropdown/Dropdown.module.scss

```diff
@@ -21,8 +21,13 @@
   padding: 8px !important;
   position: absolute;
   top: 34px;
+  visibility: hidden;
```

Comment:
I have some concerns about using visibility style. The element remains rendered and occupies some space even if popup is visible. It can hover other element and prevent it from being interactive.

Reply:
changed to display